File: Auto-generated-tasks/Task-5/Solution.cs
int Cnt = 7;
double[] T = new double[Cnt];
int i1, Itog = 1;
for (i1 = 0; i1 < Cnt; i1++) {
	if (T[i1] > T[Itog])
		Itog = i1;
}
Console.WriteLine("Itog=" + Itog);

File: Auto-generated-tasks/Task-5/Solution.py
Cnt = 7
T = [0.0] * Cnt
k1 = 0
Itog = 0
for i1 in range(0, Cnt):
	if (T[i1] > T[Itog]):
		Itog = i1
print("Itog=", Itog)